packages/big-design/src/components/InlineAlert/InlineAlert.tsx

```diff
@@ -27,12 +28,27 @@ export const InlineAlert: React.FC<InlineAlertProps> = memo(({ className, style,
 
   const renderedHeader = useMemo(() => header && <StyledHeader>{header}</StyledHeader>, [header]);
 
+  const renderedActions = useMemo(
+    () =>
+      props.actions && (
+        <Box marginTop="xSmall">
+          {props.actions.map(({ text, ...actionProps }, index) => (
+            <Button key={index} {...actionProps}>
+              {text}
+            </Button>
```

Comment:
Make sure the intended behavior is for these to go to full-width on mobile.

Reply:
👍 
<img width="386" alt="Screenshot 2020-09-23 at 12 13 12" src="https://user-images.githubusercontent.com/9980803/93992438-37b8c880-fd96-11ea-8a16-3026514ce998.png">

Reply:
I think we shouldn't stretch buttons to the full width but use their initial size even on mobile. See examples below 
![image](https://user-images.githubusercontent.com/65546214/94149113-2815ae80-fe80-11ea-9d14-ef87e54393e7.png)

Reply:
@eugene-polev I fixed it 👍 

<img width="382" alt="Screenshot 2020-09-25 at 11 26 17" src="https://user-images.githubusercontent.com/9980803/94245104-767b8980-ff22-11ea-9f00-f781dfb8ec7c.png">
<img width="386" alt="Screenshot 2020-09-25 at 11 26 40" src="https://user-images.githubusercontent.com/9980803/94245107-77142000-ff22-11ea-8da2-8e15610f5dc4.png">